### DISCUSSION TITLE
Can I change pointInstance in Maya, after instance usd file load

### DISCUSSION THREAD

**Discussion by cgTD123** [Autodesk/maya-usd#1328] [Q&A]
**Describe the bug**

**Steps to reproduce**

**Expected behavior**

**Attachments**
If applicable, add screenshots, sample files, etc to help explain your problem.

**Specs (if applicable):**
 - OS & version [e.g. Centos7]
 - Compiler & version [e.g. gcc 6.3.1]
 - Maya version [e.g. Maya 2019]
 - Maya USD commit SHA [e.g. dev at caa921c1]
 - Pixar USD commit SHA [e.g. dev at b85ddac2]

**Additional context**

**Comment by santosg87** [comment]
hi there, Sorry. I don't think i understand much of the issue. are you looking at selection? or changing the instances? I would recommend moving this to a discussion, as it seems you have a question that needs a bit more info? :)

cheers!

**Reply by kxl-adsk** [reply]
Converted to a discussion.

**Comment by cgTD123** [comment]
yeah, I would like to change one instance between them,I already seaarch a lot of information, but not clear about this.
Usdviewer can change pick mode,but point instance mode not show in AL_USD,and I test maya2022,still can't get one instance in Maya for change.Does it can work?or can't be changed in Maya?
